### PR TITLE
Upgrading sailor to 2.5.4

### PR DESCRIPTION
**Pull request by ElBabbler**
**DoD**
**Development:**
- [x] use new logger
- [x] component should be dockerised, add: "buildType":"docker" line into component.json
- [x] sailor version is changed from 2.5.1 to 2.6.1
- [x] CHANGELOG.md is updated

**QA:**
- [x] Smoke test is finished
- [ ] Regression testing is finished (optional)
- [ ] Support is asked to deploy to the production stage (app.elastic.io) and/or another required stage
- [ ] Component is actually deployed to the production stage (app.elastic.io) and/or another required stage
- [ ] Smoke after deploying to the production stage

**Smoke test checklist**
- [x] Push to platform
- [x] Retrieve sample
- [x] Retrieve sample on LA (if possible)
- [x] Run keen flow
- [x] Run realtime flow

### CODE REVIEW

**Comment by uaArsen** [comment]
Circle ci failed

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "author": "elastic.io GmbH",
   "license": "Apache License 2.0",
   "dependencies": {
-    "elasticio-sailor-nodejs": "2.5.1",
+    "elasticio-sailor-nodejs": "2.6.0",
```
Comment (uaArsen) [inline package.json]: requires node 12. Right now this component uses is 10.6

Comment (ElBabbler) [reply package.json]: Changed

File: package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-  "version": "1.1.1",
+  "version": "1.1.2",
   "description": "elastic.io webhook component",
   "scripts": {
     "pretest": "eslint spec lib verifyCredentials.js --fix",
     "test": "mocha spec --recursive"
   },
   "engines": {
-    "node": ">=10.15.0"
+    "node": "^12"
```
Comment (jhorbulyk) [inline package.json]: Why not just `12`?

Comment (ElBabbler) [reply package.json]: It's the same?

Comment (ElBabbler) [reply package.json]: Changed to 12 anyway

**Pull request merged**